packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -263,6 +267,17 @@ const NavBar = (props: TNavbarProps) => {
     [props.project]
   );
 
+  // TODO: uncomment before merging
+  // if (props.isLoading) {
+  // Render the loading navbar as long as all the data
+  // hasn't been loaded, or if the project does not exist.
+  return props.isNewNavigationEnabled ? (
+    <NavBarSkeleton isExpanded={isMenuOpen} />
+  ) : (
+    <LoadingNavBar />
+  );
```

Comment:
I noticed there is some flickering due to the feature flag evaluation. I'm not sure if I'm doing something wrong and what I can do about it, but before evaluating to `true` the flag always defaults to `false`
![2023-08-10 14 28 34](https://github.com/commercetools/merchant-center-application-kit/assets/49066275/693098ff-af3c-41a3-8115-8546bf393185)

Reply:
I think we can use the `useAdapterStatus` [hook](https://github.com/tdeekens/flopflip#useadapterstatus-adapterstatus) from `flopflip` to know about the readiness of the feature flags so we don't render anything until we have the remove value locally.

Probably something like this:
```js
import { useAdapterStatus } from '@flopflip/react-broadcast';

const ComponentWithFeatureToggle = () => {
  const isFeatureEnabled = useFeatureToggle('myFeatureToggle');
  const { isReady } = useAdapterStatus();

  if (!isConfigured) return null;
  else if (!isFeatureEnabled) <NewSkeleton />;
  else return <LegacySkeleton />;
};
```

Credits to @tdeekens for letting me know about this `flopflip` hook 🙇 .

Reply:
You're welcome. It's just

```diff
-if (!isConfigured) return null;
+if (!isReady) return null;
```

Reply:
Wowzers, TIL, thank you both for sharing 🎉